<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Expense</class>
 <widget class="QDialog" name="Expense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expense</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/image/window_icon.png</normaloff>:/image/image/window_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog{border-image: url(:/image/Background04.png) 0 0 0 0 stretch stretch;}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_backto">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>710</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #000000;
background-color:#000000;
border-radius: 15px;
color: white;
font: 700 10pt &quot;Segoe UI&quot;;
	</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 1px solid #ff99cc;
 border-radius: 10px;
color: #ff99cc;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  #3C536F;
 border-radius: 10px;
background-color: transparent;
color: #3C536F;}</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>491</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QGroupBox{font: 20pt &quot;Segoe UI&quot;;
border: 5px solid  #3C536F;
 border-radius: 10px;
/* From https://css.glass 
background: rgba(0, 0, 0, 0.14);
border-radius: 16px;
box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
backdrop-filter: blur(11.4px);
-webkit-backdrop-filter: blur(11.4px);
border: 1px solid rgba(0, 0, 0, 0.11);
color: #3C536F;}*/
QGroupBox{font: 19pt &quot;Segoe UI&quot;;
color:White;

border-radius: 15px;

background-color: #841ebc;
border: 1px solid #a23fdf;}</string>
   </property>
   <property name="title">
    <string> ADD YOUR EXPENSES</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_description">
       <property name="styleSheet">
        <string notr="true">color: rgb(60, 83, 111);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_description">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_date">
       <property name="styleSheet">
        <string notr="true">color: rgb(60, 83, 111);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_expense">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_amount">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_amount">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #000000;
background-color:#000000;
border-radius: 15px;
color: white;
font: 700 10pt &quot;Segoe UI&quot;;
	</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_category">
       <property name="styleSheet">
        <string notr="true">color: rgb(60, 83, 111);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_category">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
       <item>
        <property name="text">
         <string>Food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entertainment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Traveling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clothing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Education Equipment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>University Fees</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
